graphics/vips/Portfile

```diff
@@ -59,3 +60,6 @@ depends_lib         path:lib/pkgconfig/glib-2.0.pc:glib2 \
 
 configure.post_args-append \
                    -Dintrospection=enabled
+
+configure.cflags-append \
+                   -Wno-incompatible-function-pointer-types
```

Comment:
This flag should be used conditionally, if not for macOS 14 with Clang then at least just for Clang.
Also, better use `-Wno-error=` version. Something like:

```
platform darwin 23 {
    if {[string match *clang* ${configure.compiler}]} {
        configure.cflags-append \
                    -Wno-error=incompatible-function-pointer-types
    }
}
```

Reply:
@joostdekeijzer Also please add C++11 standard, otherwise the port is broken on some systems due to a wrong compiler being picked.
```
# meson.build: ERROR: C++ Compiler does not support -std=c++11
compiler.cxx_standard   2011
```